### DISCUSSION TITLE
How to Sync Swell to Woo-Commerce

### DISCUSSION THREAD

**Discussion by User158053** [swellstores/community#328] [Help] [answered]
Hi,
I have been advised that Swell can't be linked to Amazon or Ebay to manage stock level. However, would it be possible to connect our backend Swell to Woo-Commerce to synchronise stock levels between the two? 

Thanks in advance!

**Comment by sags95** [comment, marked as answer]
@User158053 We don't yet have a simple solution to this unfortunately. We're working on the release of Swell Apps that would allow someone to develop and distribute an app that can facilitate this. In the meantime, you would mainly be using webhooks to send stock updates to WooCommerce. To update Swell, you would use the Backend API targeting the `/products:stock` endpoint to modify inventory levels. 

https://developers.swell.is/backend-api/webhooks

**Reply by sags95** [reply]
An easier way might be to also use the our Zapier integration with WooCommerce, as I see they also have a Zap.
